samples/dashboards/sync/sync-highlight-options/demo.html

```diff
@@ -0,0 +1,30 @@
+<script src="https://code.highcharts.com/dashboards/dashboards.js"></script>
+<script src="https://code.highcharts.com/dashboards/datagrid.js"></script>
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/dashboards/modules/dashboards-plugin.js"></script>
+
+<div class="inputs">
+    <span>Chart Highlight Sync Options: </span>
+
+    <input type="checkbox" id="enabled" name="showTooltip" checked>
```

Comment:
1. The name is not unique, you have another input with the same name.
2. Can we change the name of id? `enabled` sounds not good.